src/Tests/Nop.Web.MVC.Tests/Admin/Validators/Catalog/ProductValidatorTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using FluentValidation.TestHelper;
+using Nop.Web.Areas.Admin.Models.Catalog;
+using Nop.Web.Areas.Admin.Validators.Catalog;
+using Nop.Web.MVC.Tests.Public.Validators;
+using NUnit.Framework;
+
+namespace Nop.Web.MVC.Tests.Admin.Validators.Catalog
+{
+    [TestFixture]
+    public class ProductValidatorTests : BaseValidatorTests
+    {
```

Comment:
This test class must be checking max length of Name, MetaKeywords, MetaTitle, Sku, ManufacturerPartNumber, Gtin, RequiredProductIds and AllowedQuantities filds

Reply:
sorry, but I didn't found these validations in ProductValidator class, Isn't this only to test ProductValidator class? I only found one rule for Name (NotEmpty)

Reply:
The call SetDatabaseValidationRules<Product>(dbContext) adds rules to the maximum length of the string fields